projects/core/src/lib/services/navigation-error-router.service.ts

```diff
@@ -8,24 +8,30 @@ import { filter } from 'rxjs';
   providedIn: 'root',
 })
 export class NavigationErrorRouterService {
+  private urlNavigatedToAfterError: string | null = null;
+
   constructor(router: Router, @Inject(APP_BASE_HREF) baseHref: string, @Inject(LOCALE_ID) localeId: string) {
     // Use an alternative catch-all route instead of '**', this allows libraries to add routes
     router.events.pipe(
       takeUntilDestroyed(),
       filter((event): event is NavigationError => event instanceof NavigationError),
-    ).subscribe((event: NavigationError) => {
+    ).subscribe((event) => {
       console.error('Navigation error', event);
```

Comment:
Maybe we can log the error at the end of the method? If we are doing a `navigateByUrl` then the console.error might not be needed since the user will end up on the right spot? If the error is triggered again by the same URL (so the `if` statement returns false) we then can show an error?